.docker/intelmq-full-dev/entrypoint_dev.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+/opt/dev/update.sh
+
+if [ "${LOG_MAIL_ENABLED}" = "true" ]; then
+    sed -i "s/return\ logger/### Code added to fix unexistent mail handler ###\n    mail_handler=logging.handlers.SMTPHandler(mailhost = ('${LOG_MAIL_MAILHOST}', ${LOG_MAIL_PORT}),fromaddr = '${LOG_MAIL_FROMADDR}',toaddrs = ['${LOG_MAIL_TOADDR}'],subject = '${LOG_MAIL_SUBJECT}',credentials = ${LOG_MAIL_CREDENTIALS}, secure = ${LOG_MAIL_SECURE} )\n    mail_handler.setLevel(${LOG_MAIL_LEVEL})\n    mail_handler.setFormatter(logging.Formatter(LOG_FORMAT))\n    logger.addHandler(mail_handler)\n    aux_logger = logger\n    return aux_logger\n    ### End code added to fix unexistent mail handler ###/g" /opt/intelmq/intelmq/lib/utils.py
```

Comment:
I appreciate your interest in adding an SMTP logging handler, but please add this functionality to IntelMQ directly instead of hacking it into the code during the container-build.

Reply:
We use this approach in production environments since a year ago, when we started to work with intelmq, now we understand more about the project, and we could take this path, contributing with intelmq source code instead of just build in top of it.

Reply:
We started to work in this but its a huge task! For now we remove mail handler from this repo